ambry-messageformat/src/test/java/com.github.ambry.messageformat/BlobStoreHardDeleteTest.java

```diff
@@ -138,12 +138,15 @@
     private MessageFormatInputStream getPutMessage(StoreKey key, BlobProperties blobProperties, byte[] usermetadata,
         byte[] blob, int blobSize, short blobVersion, BlobType blobType)
         throws MessageFormatException {
-      if (blobVersion == MessageFormatRecord.Blob_Version_V1) {
-        return new PutMessageFormatInputStream(key, blobProperties, ByteBuffer.wrap(usermetadata),
-            new ByteBufferInputStream(ByteBuffer.wrap(blob)), blobSize, blobType);
-      } else {
-        return new PutMessageFormatBlobV2InputStream(key, blobProperties, ByteBuffer.wrap(usermetadata),
-            new ByteBufferInputStream(ByteBuffer.wrap(blob)), blobSize, blobType);
+      switch (blobVersion) {
```

Comment:
Sorry, I dont get you. Based on your suggestion I changed this to a switch case. For other such instances, as I commented earlier, I dint feel switch case is a good fit as its just one assignment statement.